lib/ask/runtime/channel_broker.ex

```diff
@@ -825,6 +866,25 @@ defmodule Ask.Runtime.ChannelBroker do
     {:reply, :ok, end_state, timeout_from_config(config)}
   end
 
+  @impl true
+  def handle_call(
+        {:force_active_respondent, respondent, provider},
+        _from,
+        %{config: config, channel_id: channel_id} = state
+      ) do
+    Logger.debug("CHN_BRK force_active_respondent: #{inspect(binding())}")
+
+    end_state =
+      case provider do
+        "nuntium" ->
+          activate_respondent(state, respondent)
+      end
```

Comment:
I wouldn't restrict the function to Nuntium. We only call it from Nuntium but that's the channel's choice/requirement to use the feature, or not.